src/Components/CreateImageWizardV2/steps/Review/Footer/CreateDropdown.tsx

```diff
@@ -39,7 +39,9 @@ export const CreateSaveAndBuildBtn = ({
       (await createBlueprint({
         createBlueprintRequest: requestBody,
       }).unwrap()); // unwrap - access the success payload immediately after a mutation
+
```

Comment:
Just to check - are the three new lines in `CreateDropdown` and `EditDropdown` files intentional?